src/Explore.Cli/Program.cs

```diff
@@ -252,17 +261,61 @@ internal static async Task ExportSpaces(string exploreCookie, bool? verboseOutpu
                 return;
             }
 
+            var namesList = names?.Split(',')
+                .Select(name => name.Trim())
+                .ToList();
             var spaces = await spacesResponse.Content.ReadFromJsonAsync<PagedSpaces>();
             var panel = new Panel($"You have [green]{spaces!.Embedded!.Spaces!.Count} spaces[/] in explore");
             panel.Width = 100;
             panel.Header = new PanelHeader("SwaggerHub Explore Data").Centered();
+
+            // set the file name if provided
+            string fileName = "ExploreSpaces.json";
+            if (!string.IsNullOrEmpty(exportFileName))
+            {
+                // check if the file has a valid extension
+                if (!exportFileName.Contains('.'))
+                {
+                    fileName = $"{exportFileName}.json";
+                }
+                else if (exportFileName.EndsWith(".json"))

```

Comment:
Maybe change to `EndsWith(".json", StringComparison.OrdinalIgnoreCase)` to support case insensitivity